jump/src/lib.rs

```diff
@@ -56,10 +56,10 @@ boot-pack
     Pack the given lift manifests into scie executables. If no manifests
     are given, looks for `lift.json` in the current directory. By
     default the current scie-jump is used as the scie tip, but an
-    alternate scie-jump binary can be specified using --path. By default
-    the lift manifest is appended to the tail of the scie as a single
-    line JSON document, but can be made a multi-line pretty-printed JSON
-    document by passing --no-single-lift-line.
+    alternate scie-jump binary can be specified using --scie-jump. By
```

Comment:
The option processing for `boot-pack` is here:
https://github.com/a-scie/jump/blob/1bd89d3459ac0e14a02cd31443ab3ff4c8466fd3/src/boot/pack.rs#L259-L283